packages/eslint-config-pantry/index.json

```diff
@@ -0,0 +1,84 @@
+{
+  "no-restricted-imports": [
```

Comment:
Having this as a global rule instead of an override might make it so that other `no-restricted-imports` get clobbered such as these in base: https://github.com/reside-eng/lint-config/blob/9d9c634f2f7cd3ee407878fc4b583a55242c9ea2/packages/eslint-config-base/rules/imports.js#L102

Also FWIW we might be able to add these rules to https://github.com/reside-eng/lint-config/blob/9d9c634f2f7cd3ee407878fc4b583a55242c9ea2/packages/eslint-config-react/rules/base.js#L11 instead of needing to have our own package since all react codebases should be using pantry 🤔